### PR TITLE
I18N-1269 Uptick to v3.3.3 of spring boot

### PR DESCRIPTION
**Pull request by DarKhaos**
Upticked Mojito to v3.3.3 of Spring Boot

### CODE REVIEW

File: mavenplugin/pom.xml [hunk 1]
```diff
@@ -29,6 +29,7 @@
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
             <version>3.3.9</version>
+            <scope>provided</scope>
```
Comment (maallen) [inline mavenplugin/pom.xml:32]: What's the reasoning for updating the scope here?

File: webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyScreenshot.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public class ThirdPartyScreenshot extends AuditableEntity {
   @Column(name = "third_party_id")
   String thirdPartyId;
 
-  @OneToOne(fetch = FetchType.LAZY)
+  @ManyToOne(fetch = FetchType.LAZY)
```
Comment (maallen) [inline webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyScreenshot.java:26]: Why is this change required? It looks like a fundamental change in the entity relationship and not related to a library uptick

Comment (DarKhaos) [reply webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyScreenshot.java]: I made the change to fix a warning message at compile time, but I reverted it.

File: mavenplugin/pom.xml (hunk 1 above)
Comment (DarKhaos) [reply mavenplugin/pom.xml]: I made the change to fix a warning message at compile time, but I reverted it.
